torchvision/transforms/v2/functional/_utils.py

```diff
@@ -101,78 +113,44 @@ def _get_kernel(dispatcher, input_type):
             elif cls in registry:
                 return registry[cls]
 
-        # Note that in the future we are not going to return a noop here, but rather raise the error below
+    if allow_passthrough:
         return _noop
 
-    raise TypeError(
-        f"Dispatcher {dispatcher} supports inputs of type torch.Tensor, PIL.Image.Image, "
-        f"and subclasses of torchvision.datapoints.Datapoint, "
-        f"but got {input_type} instead."
-    )
-
+    supported_datapoint_types = registry.keys() - {torch.Tensor, PIL.Image.Image}
```

Comment:
I would typically leave out that kinda of fancy logic for later. It takes time to write, takes time to review, and adds risks of bugs. No need to remove it now that it's already been written, but we're close to a release and a simple `", ".join(registry.keys())` would have been more than enough.

The `complexity / added value` ratio seems extremely small.